plans/convert.pp

```diff
@@ -40,6 +43,11 @@
     $compiler_hosts,
   )
 
+  # Know what version of PE the current targets are
+  $pe_version = run_task('peadm::read_file', $master_target,
+    path => '/opt/puppetlabs/server/pe_version',
+  )[0][content].chomp
+
```

Comment:
Oh we have the PE version here already.  So my comment above should use this value.